src/EditorFeatures/CSharp/CodeCleanup/CSharpCodeCleanupService.cs

```diff
@@ -62,7 +62,7 @@ internal class CSharpCodeCleanupService : AbstractCodeCleanupService
                 // Expression-level preferences
                 //   dotnet_style_coalesce_expression
                 new DiagnosticSet(FeaturesResources.Apply_coalesce_expression_preferences,
-                    IDEDiagnosticIds.UseCoalesceExpressionDiagnosticId),
+                    IDEDiagnosticIds.UseCoalesceExpressionForTernaryConditionalCheckDiagnosticId),
```

Comment:
Shouldn't all coalesce expression related diagnostic IDs be passed here?

Reply:
I'm not changing the behavior of this system.  I've passed along the feedback that it is broken as it requires a separate map to ensure all these connections.